conf/modules.config

```diff
@@ -126,26 +127,47 @@ process {
     }
 
     withName: GROHMM_PARAMETERTUNING {
-        ext.when = { params.transcript_identification == "grohmm" && !params.skip_tuning }
+        ext.when = { !params.skip_grohmm && !params.skip_tuning && params.assay_type == "GROseq" }
     }
 
     withName: GROHMM_TRANSCRIPTCALLING {
-        ext.when = { params.transcript_identification == "grohmm" }
+        ext.when = { !params.skip_grohmm && params.assay_type == "GROseq" }
     }
 
     withName: HOMER_FINDPEAKS {
-        ext.when = { params.transcript_identification == "homer" }
-        ext.args = "-style factor"
+        ext.when = { params.assay_type == "GROseq" }
+        // FIXME Handle other assays
```

Comment:
TODO is more typical, or do you want to distinguish with nf-core TODO?

Reply:
Ah, it's the next line, I have the `-style groseq` hardcoded. Outside the scope, but I agree, it's a todo!